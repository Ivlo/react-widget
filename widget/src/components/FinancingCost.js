import React from "react";

const FinancingCost = () => {
  return (
    <>
      <header>
        <h1>Sequra</h1>
        <h2>Fracciona tu pago</h2>
      </header>
      <ol>
        <li>Eliges Fracciona tu pago al realizar tu pedido y pagas solo la primera cuota.</li>
        <li>Recibes tu pedido</li>
        <li>El resto de pagos se cargaran automáticamente a tu tarjeta</li>
      </ol>
      <p>¡Así de simple!</p>
      <p>
        Además en el importe mostrado ya se incluye la cuota única mensual de 3,00 € al mes, por lo
        que no tendrás ninguna sorpresa
      </p>
    </>
  );
};

export default FinancingCost;
