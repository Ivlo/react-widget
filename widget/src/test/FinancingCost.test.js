import React from "react";
import { shallow } from "enzyme";
import FinancingCost from "../components/FinancingCost";

const wrapper = shallow(<FinancingCost />);

describe("FinancingCost", () => {
  it("should render the FinancingCost Component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
