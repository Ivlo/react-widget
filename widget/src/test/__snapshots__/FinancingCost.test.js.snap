// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FinancingCost should render the FinancingCost Component correctly 1`] = `
<Fragment>
  <div
    className="heading"
  >
    <h1>
      Fracciona tu pago
    </h1>
    <img
      alt="Sequra"
      src="https://assets-global.website-files.com/5f5f72f9edda90fbda5af283/5fd274a0a7dd297f88c5d8f2_SeQura-0.svg"
    />
  </div>
  <ol
    className="financing-steps"
  >
    <li
      className="financing-steps__item"
    >
      <p
        className="financing-steps__item__text"
      >
        Eliges Fracciona tu pago al realizar tu pedido y pagas solo la primera cuota.
      </p>
      <img
        alt="financing icon"
        src="https://assets-global.website-files.com/5f5f72f9edda90fbda5af283/5f71d32fcb2fc156704d4689_picto_formacion.svg"
      />
    </li>
    <li
      className="financing-steps__item"
    >
      <p
        className="financing-steps__item__text"
      >
        Recibes tu pedido
      </p>
      <img
        alt="financing icon"
        src="https://assets-global.website-files.com/5f5f72f9edda90fbda5af283/5f71d32fcb2fc156704d4689_picto_formacion.svg"
      />
    </li>
    <li
      className="financing-steps__item"
    >
      <p
        className="financing-steps__item__text"
      >
        El resto de pagos se cargaran automáticamente a tu tarjeta
      </p>
      <img
        alt="financing icon"
        src="https://assets-global.website-files.com/5f5f72f9edda90fbda5af283/5f71d32fcb2fc156704d4689_picto_formacion.svg"
      />
    </li>
  </ol>
  <p>
    <strong>
      ¡Así de simple!
    </strong>
  </p>
</Fragment>
`;
